pom.xml

```diff
@@ -39,7 +50,7 @@
 		<dependency>
 			<groupId>info.bliki.wiki</groupId>
 			<artifactId>bliki-core</artifactId>
-			<version>3.0.16</version>
+			<version>3.1.0-SNAPSHOT</version>
```

Comment:
This might potentially be dangerous since bliki is being built on a frequent basis, but maybe we can specify a build version and upgrade ocasionally ?